src/main/java/com/spark/masterdata/team/controller/TeamController.java

```diff
@@ -1,9 +1,36 @@
 package com.spark.masterdata.team.controller;
 
+import com.spark.masterdata.team.dto.TeamDTO;
+import com.spark.masterdata.team.entity.Team;
+import com.spark.masterdata.team.service.TeamService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
 import org.springframework.web.bind.annotation.RestController;
 
+import javax.validation.Valid;
+import java.util.List;
+
 @RestController
 public class TeamController {
+    @Autowired
```

Comment:
use constructor injection as per spring boot standard
private final TeamService teamService;

public TeamController(TeamService teamService) {
     this.teamService = teamService;
}